setup.sh

```diff
@@ -13,20 +14,26 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
 	exit 2
 fi
 
+if [ ! -e ${GROVEPI_DIR} ]; then
+	echo "Error: GrovePi directory not found."
+	exit 2
```

Comment:
This should probably be a different error code than the previous error

Reply:
Fixing.

Reply:
I didn't see any Unix error codes that were particularly meaningful for us, so I just incremented them so at least code inspection will point you to the right place.